cplusplus.json

```diff
@@ -691,63 +700,63 @@
+  {"id": "cpp.restbed", "name": "Restbed", "tech": ["cpp-web"], "tags": ["rest"], "repo": "corvusoft/restbed",
   "imports": ["restbed"], "examples": ["https://github.com/Corvusoft/restbed/tree/master/test"]},
   {"id": "cpp.cpp-netlib", "name": "C++ Network Library", "tech": ["networking"], "tags": ["library"], "repo": "cpp-netlib/cpp-netlib",
   "imports": ["network/"], "examples": ["https://github.com/cpp-netlib/cpp-netlib/tree/master/libs/network/example"]},
   {"id": "cpp.cpr", "name": "C++ Requests", "tech": ["internet"], "tags": ["curl"], "repo": "whoshuu/cpr",
   "imports": ["cpr/cpr.h"], "examples": ["https://github.com/whoshuu/cpr/tree/master/test"]},
-  {"id": "cpp.curlcpp", "name": "curlcpp", "tech": ["internet"], "tags": ["curl"], "repo": "JosephP91/curlcpp",
+  {"id": "cpp.curlcpp", "name": "curlcpp", "tech": ["internet"], "tags": ["curl-wrapper"], "repo": "JosephP91/curlcpp",
   "imports": ["curlcpp/"], "examples": ["https://github.com/JosephP91/curlcpp#simple-usage-example"]},
   {"id": "cpp.dyad-c", "name": "Dyad.c", "tech": ["networking"], "tags": ["async"], "repo": "rxi/dyad",
   "imports": ["dyad.h"], "examples": ["https://github.com/rxi/dyad/tree/master/example"]},
   {"id": "cpp.enet", "name": "ENet", "tech": ["networking"], "tags": ["udp"], "repo": "lsalzman/enet",
   "imports": ["enet/enet.h"], "examples": ["https://github.com/lsalzman/enet"]},
-  {"id": "cpp.evpp", "name": "evpp", "tech": ["internet"], "tags": ["multiprotocol"], "repo": "Qihoo360/evpp",
+  {"id": "cpp.evpp", "name": "evpp", "tech": ["networking"], "tags": ["multiprotocol"], "repo": "Qihoo360/evpp",
   "imports": ["evpp/"], "examples": ["https://github.com/Qihoo360/evpp/tree/master/examples"]},
-  {"id": "cpp.http-parser", "name": "HTTP Parser", "tech": ["app-data"], "tags": ["http-parser"], "repo": "nodejs/http-parser",
+  {"id": "cpp.http-parser", "name": "HTTP Parser", "tech": ["internet"], "tags": ["http-parser"], "repo": "nodejs/http-parser",
   "imports": ["http_parser.h"], "examples": ["https://github.com/nodejs/http-parser/blob/master/test.c"]},
   {"id": "cpp.libcurl", "name": "libcurl", "tech": ["internet"], "tags": ["curl"], "repo": "curl/curl",
   "imports": ["curl/"], "examples": ["https://curl.haxx.se/libcurl/c/allexamples.zip"]}
   {"id": "cpp.quic", "name": "QUIC", "tech": ["networking"], "tags": ["udp"], "repo": "devsisters/libquic",
   "imports": ["quic_"]},
   {"id": "cpp.libwebsockets", "name": "libwebsockets", "tech": ["networking"], "tags": ["sockets"], "repo": "warmcat/libwebsockets",
   "imports": ["libwebsockets.h"], "examples": ["https://github.com/warmcat/libwebsockets/tree/master/minimal-examples"]},
-  {"id": "cpp.mongoose", "name": "mongoose", "tech": ["internet"], "tags": ["embedded"], "repo": "cesanta/mongoose",
+  {"id": "cpp.mongoose", "name": "mongoose", "tech": ["cpp-web"], "tags": ["web-server"], "repo": "cesanta/mongoose",
```

Comment:
I don't agree with cpp-web. I think it should have stayed as 'internet'. The logic here is XXX-web is to build your web apps using a certain framework. Embedded servers are rarely used for web apps. The usually have a simple utilitarian purpose, such as 'provide status of 3d printer' or something. cpp-web would be a good category to describe CGI programs made in C++, for example.